src/components/Dashboard/TodoList.vue

```diff
@@ -78,10 +94,10 @@
                   </div>
                 </div>
 
-                <div v-else-if="isError(item)" class="todo-status">
+                <div v-else-if="isError(task)" class="todo-status">
                   <div>FILING PENDING</div>
                   <div class="vert-pipe">&nbsp;</div>
-                  <div class="payment-status" v-if="inProcessFiling !== undefined && inProcessFiling === item.id">
+                  <div class="payment-status" v-if="inProcessFiling !== undefined && inProcessFiling === task.id">
```

Comment:
This is not your code but I believe you don't need the first condition in the v-if -- if the second condition is true then the first condition must also be true. Please update (and test) if time allows.

Reply:
K, will change this.